cpp/src/utilities/device_atomics.cuh

```diff
@@ -942,7 +942,17 @@ T atomicCAS(T* address, T compare, T val)
     return cudf::detail::typesAtomicCASImpl<T, sizeof(T)>()(address, compare, val);
 }
 
-
+/**
+ * @overload cudf::bool8 atomicCAS(cudf::bool8* address, cudf::bool8 compare, cudf::bool8 val)
+ */
+__forceinline__ __device__
+cudf::bool8 atomicCAS(cudf::bool8* address, cudf::bool8 compare, cudf::bool8 val)
```

Comment:
`atomicCAS` looks good to me.
Why don't we have `atomicAnd`, `atomicOr`, `atomicXOR` for `cudf::bool8`?

Reply:
@kovaltan see https://github.com/rapidsai/cudf/pull/1142#issuecomment-485625722